components/director/internal/repo/exist_test.go

```diff
@@ -0,0 +1,67 @@
+package repo_test
+
+import (
+	"context"
+	"regexp"
+	"testing"
+
+	"github.com/kyma-incubator/compass/components/director/internal/persistence"
+	"github.com/kyma-incubator/compass/components/director/internal/repo"
+	"github.com/kyma-incubator/compass/components/director/internal/repo/testdb"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/require"
+)
+
+func TestExist(t *testing.T) {
+	givenID := uuidA()
+	givenTenant := uuidB()
+	givenQuery := "SELECT 1 FROM users WHERE tenant_col = $1 AND id_col = $2"
+	escapedQuery := regexp.QuoteMeta(givenQuery)
+	sut := repo.NewExistQuerier("users", "tenant_col")
+
+	t.Run("when exist", func(t *testing.T) {
```

Comment:
What do you think about this test name:`success when exist`. It would be compliant with the rest of our tests.